<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Recipes</class>
 <widget class="QMainWindow" name="Recipes">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>808</width>
    <height>388</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Recipes</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="recipe_list_tablewidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>791</width>
      <height>341</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Name</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Description</string>
     </property>
    </column>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>808</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuManage">
    <property name="title">
     <string>Manage</string>
    </property>
    <addaction name="actionSave"/>
    <addaction name="actionNew"/>
   </widget>
   <addaction name="menuManage"/>
  </widget>
  <action name="actionSave">
   <property name="text">
    <string>Add</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>Remove</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
